composeApp/src/commonMain/kotlin/org/ooni/probe/di/Dependencies.kt

```diff
@@ -9,17 +11,21 @@ import org.ooni.engine.NetworkTypeFinder
 import org.ooni.engine.OonimkallBridge
 import org.ooni.engine.TaskEventMapper
 import org.ooni.engine.models.NetworkType
+import org.ooni.probe.data.SettingsRepository
 import org.ooni.probe.data.models.TestResult
 import org.ooni.probe.shared.PlatformInfo
 import org.ooni.probe.ui.dashboard.DashboardViewModel
 import org.ooni.probe.ui.result.ResultViewModel
 import org.ooni.probe.ui.results.ResultsViewModel
+import org.ooni.probe.ui.settings.SettingsViewModel
+import org.ooni.probe.ui.settings.category.SettingsCategoryViewModel
 
 class Dependencies(
     val platformInfo: PlatformInfo,
     private val oonimkallBridge: OonimkallBridge,
     private val baseFileDir: String,
     private val cacheDir: String,
+    private val dataStore: DataStore<Preferences>,
```

Comment:
It would be nice if this was a factory instead. This way we're creating the DataStore when the app starts up, when we could be creating it just when it really needs to be used. Maybe it will be used soon anyway, but we should delay as much as we can.

Also, we should try to create the datastore in a background coroutine, since it's a disk read/write operation.